docs/fdc3-glossary.md

```diff
@@ -6,24 +6,29 @@ sidebar_label: Glossary
 
 For the purposes of this Standard, the following definitions apply. Other terms are defined when first used, at which place they appear in bold and italic type. Terms explicitly defined in this Standard are not to be presumed to refer implicitly to similar terms defined elsewhere. Terms not defined are assumed to be well-known in the financial services or software industries.
 
-
+- **Agent Bridge**: Shorthand for Desktop Agent Bridge.
```

Comment:
What is the convention for using capitals vs. lower case for terms in the glossary?

Reply:
Rex wrote it down somewhere, will have to hunt for it. Basically, capitalize references to specific definitions of a thing provided by 'the Standard' (also capitalized, along with a Part of the Standard), e.g. "Desktop Agent"
but use lower-case where they are used as a class of thing (i.e. plural), e.g. "desktop agents".